### ISSUE TITLE
No setup information found for ghc-8.4.3 on FreeBSD

### ISSUE THREAD

**Issue by yzgyyang** [commercialhaskell/stack#4169]
### General summary/comments (optional)

```
vagrant@freebsd:~ % stack setup
Writing implicit global project config file to: /home/vagrant/.stack/global-project/stack.yaml
Note: You can change the snapshot via the resolver field there.
Using latest snapshot resolver: lts-12.2
Downloaded lts-12.2 build plan.
No setup information found for ghc-8.4.3 on your platform.
This probably means a GHC bindist has not yet been added for OS key 'freebsd64'.
Supported versions: ghc-7.8.4, ghc-7.10.1, ghc-7.10.2, ghc-7.10.3, ghc-8.0.1, ghc-8.0.2, ghc-8.2.1, ghc-8.2.2, ghc-8.4.1, ghc-8.4.2
```
### Actual

See error above.

### Stack version

```
vagrant@freebsd:~ % stack --version
Version 1.7.1, Git revision 681c800873816c022739ca7ed14755e85a579565 (5807 commits) x86_64 hpack-0.28.2
```

### Method of installation

```
curl -fsSL -o haskellstack.sh https://get.haskellstack.org/
chmod +x haskellstack.sh
./haskellstack.sh
stack setup
```

**Comment by borsboom** [comment]
It looks like no FreeBSD GHC bindists were provided for 8.4.3.  See https://www.haskell.org/ghc/download_ghc_8_4_3.html#freebsd11_x86_64.

When FreeBSD bindists are made available, we can add them to the `stack setup` metadata.

@bgamari, was leaving out FreeBSD bindists for this release intentional?

**Comment by bgamari** [comment]
Yes, unfortunately GHC 8.4.3 is unbuildable on FreeBSD is broken due to https://github.com/haskell/unix/issues/102. This will be fixed in 8.6. Sorry for the inconvenience!

**Comment by borsboom** [comment]
Thanks for the info.  I'll close this issue, since there's nothing we can do about it (and we'll add freebsd bindists for 8.6 when they're available as a matter of course).
